pkg/operator/ceph/nfs/spec.go

```diff
@@ -185,7 +184,7 @@ func (r *ReconcileCephNFS) connectionConfigInitContainer(nfs *cephv1.CephNFS, na
 			keyring.VolumeMount().Resource(instanceName(nfs, name)),
 		},
 		nfs.Spec.Server.Resources,
-		mon.PodSecurityContext(),
+		controller.PodSecurityContext(),
```

Comment:
Is there any reason or ability for us to extend the log collector/rotator to NFS daemons?

Reply:
I'll look into that.

Reply:
Unfortunately for us, the Ganesha daemon can either log to `STDERR` or on files. So if we log on file with the collector we won't see anything on the container's log.
There is another hacky approach where we can implement a different side-car logger container. In this different side-car logger we could read the process STDERR's fd and pipe it to a file and then rotate. Unfortunately, we don't have the tooling for this in our container image...

So for now, we might just live this one on the side...

Reply:
Hmm... we could also consider a sidecar that perpetually tails the log file?

Reply:
Yes, but it can be a bit confusing for users to see nothing if they request the logs from the main container... They will have to describe the pod and see all the containers and find the one that logs... Not a great experience IMO.